SDK/AppCenter/Microsoft.AppCenter/Microsoft.AppCenter.csproj

```diff
@@ -15,6 +15,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="MSBuild.Sdk.Extras" Version="1.1.0" PrivateAssets="All" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
This new dependency needs to be discussed.

Reply:
Also that's the cause of the build failure. It's not referenced correctly everywhere it's needed.